CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to the "vscode-iot-device-cube" extension will be documented
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
+## Version 0.1.2
+
+### Added
+By coorperating with Azure IoT Device Workbench's dependency [vscode-iot-device-cube-sdk (version 0.0.21)](https://www.npmjs.com/package/vscode-iot-device-cube-sdk), IoT Device Cube extension now is able to help Azure IoT Device Workbench extension to upload folder from container to device such as Raspberry Pi, and to config DevKit in a container. Related changes are listed as follows.
```

Comment:
### Fixed

- When in remote mode, upload folder from workspace file system to target device through ssh.